src/Interpreter/ExternalFuncs/ChainCall.cpp

```diff
@@ -0,0 +1,58 @@
+#include "caffeine/Interpreter/ExternalFuncs/ChainCall.h"
+#include "caffeine/Interpreter/ExternalFunction.h"
+#include "caffeine/Interpreter/InterpreterContext.h"
+#include "caffeine/Support/LLVMFmt.h"
+#include <boost/range/algorithm/find_if.hpp>
+#include <fmt/format.h>
+#include <iterator>
+#include <llvm/IR/InstrTypes.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/IR/Module.h>
+
+#include <iostream>
+
+#include "unwind.h"
+
+namespace caffeine {
+
+ChainCall::ChainCall(std::vector<llvm::Function*> funcs)
+    : ExternalStackFrameMixin({}, nullptr), funcs(funcs) {}
```

Comment:
This should be a rvalue reference
```suggestion
ChainCall::ChainCall(std::vector<llvm::Function*&&> funcs)
    : ExternalStackFrameMixin({}, nullptr), funcs(std::move(funcs)) {}
```